### PR TITLE
design changes and new page for each pokemon

### PR DESCRIPTION
**Pull request by arunsahoo-xt**
Made the page responsive ,resized the image and added a information page for each pokemon

### CODE REVIEW

**Review by nishantc7 — CHANGES_REQUESTED** [review]
Change this back to the previous homepage and script. You can do this change on your forked repo.

File: package.json [hunk 1]
```diff
@@ -2,23 +2,25 @@
   "name": "monsters-rolodex",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://nishantc7.github.io/simple-pokedex",
```
Comment (nishantc7) [inline package.json]: Change this back to the previous homepage and script. You can do this change on your forked repo.

File: package.json [hunk 2]
```diff
@@ -2,23 +2,25 @@
...
+    "react-router-dom": "^6.3.0",
     "react-scripts": "3.4.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "predeploy": "yarn build",
+    "predeploy": "npm run build",
```
Comment (nishantc7) [inline package.json]: this too.

File: src/App.js [hunk 3]
```diff
@@ -12,11 +14,25 @@ class App extends Component {
...
+    
+  //   .then(results => results['results'])
+  //   .then(users => this.setState({monsters: users}))
+  // }
+
+  componentDidMount=async()=> {
+    try{
+    const response=await fetch('https://pokeapi.co/api/v2/pokemon?limit=500');
+    const data=await response.json();
+    console.log(response);
```
Comment (nishantc7) [inline src/App.js]: remove console.logs

File: src/components/card-list/card-list.component.jsx [hunk 4]
```diff
@@ -1,10 +1,38 @@
...
 export const CardList  = props => {
-    return <div className="card-list">{
+    const [user, setUser] = useState({"yo":"hehe"});
+    const UserContext = createContext()
+  const navigate=useNavigate();
+    const handleAction=async(ele)=>{
+        const response=await fetch(ele);
+const data=await response.json();
+
+console.log(data)
```
Comment (nishantc7) [inline src/components/card-list/card-list.component.jsx]: Remove console.log

File: src/components/card-list/card-list.component.jsx [hunk 5]
```diff
@@ -1,10 +1,38 @@
...
+import PokeInfo from '../solo-card/Poke-info';
+import {useNavigate } from 'react-router-dom';
 export const CardList  = props => {
-    return <div className="card-list">{
+    const [user, setUser] = useState({"yo":"hehe"});
+    const UserContext = createContext()
+  const navigate=useNavigate();
+    const handleAction=async(ele)=>{
+        const response=await fetch(ele);
+const data=await response.json();
```
Comment (nishantc7) [inline src/components/card-list/card-list.component.jsx]: Use prettier or eslint

File: src/components/card-list/card-list.component.jsx [hunk 6]
```diff
@@ -1,10 +1,38 @@
...
+                // <div class="box">
+                <div className='poke-tile' >     
+            <Card key={monster.id} monster={ monster }/>
+            </div>
+            //  <div class="details">
+            //     <div class="content">
+            //         <h1>Samantha</h1>
+            //     </div></div></div> 
+            ))
+      }
```
Comment (nishantc7) [inline src/components/card-list/card-list.component.jsx]: Remove comments

**Review by nishantc7 — CHANGES_REQUESTED** [review]

**Comment by nishantc7** [comment]
How do I view the new information page, clicking on any pokemon doesnt open a new page or modal

**Comment by nishantc7** [comment]
Great work on the stats page and responsiveness 🎉

File: src/components/card-list/card-list.component.jsx (hunk 5 above)
Comment (arunsahoo-xt) [reply src/components/card-list/card-list.component.jsx]: is it for format document or something else?

Comment (nishantc7) [reply src/components/card-list/card-list.component.jsx]: Yes for formatting.

Comment (arunsahoo-xt) [reply src/components/card-list/card-list.component.jsx]: I can do that using vs code formatting options i guess.

**Comment by arunsahoo-xt** [comment]
I made the changes.Please review it

**Comment by nishantc7** [comment]
Great work @arunsahoo-xt :tada:  :+1:

**Pull request merged**